### DISCUSSION TITLE
momo acrobatic jump

### DISCUSSION THREAD

**Discussion by worknlife** [Monitor221hz/Pandora-Behaviour-Engine-Plus#148] [Patches]
just wanted to know if a patch for [momo acrobatic jump ](https://www.nexusmods.com/skyrimspecialedition/mods/32189) can be added.
thanks

**Comment by lobotomy-x** [comment]
Explain why you need a patch, because as far as I can tell this should just work normally, though it may need the patch that comes bundled with nemesis. Did you actually try to patch it? If so and it failed you need to read the readme again. But this is an old deprecated mod tbh. You should just use jump behavior overhaul + freerun in skyrim which is designed to be similar but more modern https://www.nexusmods.com/skyrimspecialedition/mods/52865
